lib/api/controllers/baseController.js

```diff
@@ -572,6 +572,29 @@ class BaseController {
     return id;
   }
 
+  /**
+   * Returns an array of ids specified in the request
+   *
+   * @param {Request} request
+   *
+   * @throws
+   * @returns {String}
+   */
+  getIds(request) {
+    const id = [];
+    id.push(request.input.resource._id);
+    // if (!id) {
+    //   assertionError.throw('missing_argument', '_id');
+    // }
+
+    // if (typeof id !== 'string') {
+    //   assertionError.throw('invalid_type', '_id', 'string');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACommentedCode)